src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -363,13 +364,60 @@ public void run(final RunNotifier notifier) {
             statement.evaluate();
         } catch (AssumptionViolatedException e) {
             testNotifier.addFailedAssumption(e);
-        } catch (StoppedByUserException e) {
-            throw e;
         } catch (Throwable e) {
-            testNotifier.addFailure(e);
```

Comment:
As it was said, updating `EachTestNotifier` with these changes will make JUnit ignoring `StoppedByUserException` thrown manually from test method.

But, in this case, we can distinguish `StoppedByUserException` thrown from test method by checking `Description.isTest()` inside `EachTestNotifier`. Looks like it can be an appropriate solution.
@kcooney Are we ok with that approach?